### ISSUE TITLE
Train on my own data set

### ISSUE THREAD

**Issue by zzzzyq-18** [huanglianghua/siamfc-pytorch#60]
I used the pre-trained model you provided to try to train on my own data set, the effect was poor, and I observed that the loss was gradually increasing. What is the reason for this
